packages/airnode-protocol/credentials.example.json

```diff
@@ -77,6 +88,26 @@
       "chainId": 43113,
       "url": "https://api.avax-test.network/ext/bc/C/rpc"
     },
+    "bobaAvalanche": {
+      "accounts": { "mnemonic": "" },
+      "chainId": 43288,
+      "url": "https://replica.avax.boba.network/"
+    },
+    "bobaBNB": {
+      "accounts": { "mnemonic": "" },
+      "chainId": 56288,
+      "url": "https://replica.bnb.boba.network/"
+    },
+    "bobaEthereum": {
+      "accounts": { "mnemonic": "" },
+      "chainId": 288,
+      "url": "https://lightning-replica.boba.network/"
+    },
+    "bobaMoonbeam": {
+      "accounts": { "mnemonic": "" },
+      "chainId": 1294,
+      "url": "https://replica.bobabeam.boba.network/"
+    },
```

Comment:
Please use kebab-case here and in the directory names. So `boba-avalanche`, `boba-bnb`, etc. so that we can standardize this https://github.com/api3dao/airnode/issues/1575